vita3k/gui/src/game_context_menu.cpp

```diff
@@ -164,6 +192,51 @@ void game_context_menu(GuiState &gui, HostState &host, bool *selected, const std
                 ImGui::EndPopup();
             } else
                 ImGui::PopStyleColor();
+
+            // Manual Popup
+            if (manuals_popup)
+                ImGui::OpenPopup("Manual");
+            const auto display_size = ImGui::GetIO().DisplaySize;
+            ImGui::SetNextWindowPos(ImVec2(-5.0f, -5.0f), ImGuiCond_Always);
+            ImGui::SetNextWindowSize(ImVec2(display_size.x + 10.0f, display_size.y + 10.0f), ImGuiCond_Always);
```

Comment:
I would make the `-5.0f` and `10.0f` constexpr and global. Their own variables basically.

Reply:
i have set this for can get fullscreen and not get empty space in left/right border
not sur need create const value just for one call 🤔

Reply:
There are only advantages to constexpr- but I see what you mean. If you don't want to do it that's alright imo

Reply:
yes ^^